reduce further and document better support level of single integrations (#275)
open-quantum-safe/oqs-demos: README.md

## Team

> The Open Quantum Safe project has been taken over by LinuxFoundation within [PQCA](https://pqca.org). 

Reviewer: ```suggestion
The Open Quantum Safe project is part of the Linux Foundation within [PQCA](https://pqca.org). 
```

Author: This change would completely eliminate the history of OQS as a free/"meritocratic", non-LF project. If you don't like the term "taken over" what about "After 7 years of stewardship by UWaterloo, control of the project changed to the newly established PQCA, a Linux Foundation alliance."?

Reviewer: This is documented to my satisfaction at https://openquantumsafe.org/news/, so I don't have the need to repeat it in each README.  But if we do want to include it in each README, then I'd just copy the text from the website.

Author: > then I'd just copy the text from the website

Sounds a bit 20th century :-) What about instead of the text simply linking (actually from/for all sub project's README's) to there? This may actually remove lots of redundant text (e.g., regarding funding) in many sub projects.

Reviewer: Sure, we can do that.

Author: Added link to OQS news.

## Acknowledgments

> Most effort in this project has been provided by individual contributors working in their own time and out of personal interest to see how PQ crypto integrates into existing software stacks.

Reviewer: If you would like the README to include a statement acknowledging individual contributors, that's fine. However this statement goes beyond that in ways that I do not think should appear in the README.

Author: In which way? How would you rephrase?

Reviewer: How about:
```suggestion
This project has benefited substantially from the efforts of individual volunteer contributors.
```

Author: This text proposal is a true statement about practically any OSS project -- and it doesn't mention the "full software stack" aspect that makes/made this work particularly challenging; your wording could equally well be applied to all OQS sub projects or be left away entirely.

And for the avoidance of doubt: I do not want to stress the aspect of volunteerism (I presume meaning people working for free/outside of employment) but of individuals going beyond their assigned tasks; this for example also includes people like @xvzcf and practically all external contributors -- of whom I don't know their employment status -- but do know about (and wanted to acknowledge with my proposal) their personal interest and commitment.

Reviewer: ```suggestion
This project has benefited substantially from the efforts of individual contributors motivated to see how PQ crypto integrates into existing software stacks.
```

Author: I still see a marked difference between the wording "benefited substantially from" and "most effort has been provided" (by individual contributors): Here's the count of PRs done by "individual contributors" and contributors doing this as part of their employment:

145 to 36 contributions -- provided by 16 individuals to 4 employed ones. Thus, in both counts about 80% has been done by individual contributors, in my eyes justifying the wording I proposed.

Reviewer: Resolve this as you see fit.

Author: Done
